.prow.yaml

```diff
@@ -7,7 +7,7 @@ presubmits:
       preset-goproxy: "true"
     spec:
       containers:
-        - image: ghcr.io/kcp-dev/infra/build:1.23.7-2
+        - image: ghcr.io/kcp-dev/infra/build:1.24.4-1
```

Comment:
We just pushed 1.24.5-1, can we maybe use that? https://github.com/kcp-dev/infra/pkgs/container/infra%2Fbuild/472022106?tag=1.24.5-1